modules/hivemq-edge-module-opcua/src/main/java/com/hivemq/edge/adapters/opcua/mqtt2opcua/BuiltinJsonSchema.java

```diff
@@ -195,7 +195,7 @@ public static void populatePropertiesForBuiltinType(
                 return;
             case Double:
                 nestedPropertiesNode.set("type", new TextNode("number"));
-                nestedPropertiesNode.set(MINIMUM_KEY_WORD, new DoubleNode(Double.MIN_VALUE));
+                nestedPropertiesNode.set(MINIMUM_KEY_WORD, new DoubleNode(-Double.MAX_VALUE));
```

Comment:
Why can you simply say -double max?

Reply:
because of the IEEE specification. They habe a sign for the bit.